pkg/controller/lib.go

```diff
@@ -96,6 +96,30 @@ func (c *Controller) CreateStatefulSetPodDisruptionBudget(sts *apps.StatefulSet)
 	return err
 }
 
+func (c *Controller) CreateStatefulSetPDBWithCustomLabelSelectors(sts *apps.StatefulSet, labels map[string]string, selectors map[string]string) error {
```

Comment:
Remove this. Instead add `SyncStatefulSetPodDisruptionBudgetWIthLabelSelectors`. Use this function inside of `SyncStatefulSetPodDisruptionBudget` to minimize code duplication.

Reply:
`Create` -> `Sync` done. 
There is a little change in `maxUnavailable` count in the customSelector func. Thats why I hadn't try to minimize code duplication. 
Can you check this now..